doc/api/typescript.md

```diff
@@ -68,6 +68,19 @@ By intentionally not supporting syntaxes that require JavaScript code
 generation, and by replacing inline types with whitespace, Node.js can run
 TypeScript code without the need for source maps.
 
+The suggested `tsconfig.json` settings for type stripping are:
+
+```json
+{
+  "compilerOptions": {
+     "module": "nodenext",
```

Comment:
```suggestion
     "target": "esnext", 
     "module": "nodenext",
```

Amaro does not perform down-levelling (compiling new JS features into older JS). This ensures the TS output matches that expectation.

Reply:
I wish TS would change the default behavior to target esnext. Maybe one day.